web/tests/driver.py

```diff
@@ -60,8 +60,8 @@ def ExpectAttributeEqual(self, path, attribute_name, value):
     attribute_value = element.get_attribute(attribute_name)
     assert(attribute_value == value)
 
-  def ExpectContains(self, path, needle, should_exist=True):
-    element = self.FindElement(path)
+  def ExpectContains(self, path, needle, should_exist=True, check_visible=True):
+    element = self.FindElement(path, should_exist=should_exist, check_visible=check_visible)
```

Comment:
If you're trying to check that an element doesn't contain some text it will throw an exception if the element exists (even if that text isn't present). I think it would work if you take out should_exist=should_exist.